src/main/java/com/faforever/client/chat/ChatUserItemController.java

```diff
@@ -314,9 +315,11 @@ public void onMouseEnterUsername() {
     Tooltip userTooltip = new Tooltip();
     usernameLabel.setTooltip(userTooltip);
     userTooltip.textProperty().bind(Bindings.createStringBinding(
-        () -> i18n.get("userInfo.ratingFormat", getGlobalRating(player), getLeaderboardRating(player)),
+        () -> String.format("%s\n%s",
+            i18n.get("userInfo.ratingFormat", getGlobalRating(player), getLeaderboardRating(player)),
+            i18n.get("userInfo.idleTimeFormat", Duration.between(player.getIdleSince(), Instant.now()).toMinutes())),
```

Comment:
If you want an implementation that not only supports minutes ("Idle since: 224 min") you can either use `com.faforever.client.util.TimeService#shortDuration` to build a string like "Idle since: 30min 42s" or "Idle since: 3h 44min"
or `com.faforever.client.util.TimeService#timeAgo` to build a "Last active: 3h 44min ago".

If you don't want that, I do :P

Reply:
agreed ;), didn't see someone already put work into this.

Reply:
That's what I'm here for :) I don't expect everyone to know everything that exists in the client.